examples/watch/Program.cs

```diff
@@ -13,8 +13,8 @@ private static void Main(string[] args)
 
             IKubernetes client = new Kubernetes(config);
 
-            var podlistResp = client.ListNamespacedPodWithHttpMessagesAsync("default", watch: true).Result;
-            using (podlistResp.Watch<V1Pod>((type, item) =>
+            var podlistResp = client.ListNamespacedPodWithHttpMessagesAsync("default", watch: true);
+            using (podlistResp.WatchAsync<V1Pod, V1PodList>((type, item) =>
```

Comment:
do you want 
```
using (await ...)
```

here using is to dispose Task

Reply:
I don't think so, because the whole goal was to print the 'ctl-c to cancel' before the wait completes, but perhaps I should drop the 'using' part..

Reply:
`await WatchAsync` creates a new Watcher which will not block running to code inside `using`

Reply:
fwiw, you can't use await here because then the Main would have to be async and dotnet doesn't like that. I tried using `.Result` but it hangs w/o printing the `ctrl-c` stuff.

I'm open to suggestions, but for now I think this is the best we can do (or we could just remove the `using` syntax...)

Reply:
Actually, I just rewrote the code to avoid the `using` directive and to call `Dispose` directly.

Reply:
sorry I finally understand the issue I did not look into #301 carefully before
please see #313 for my proposal